J/JSBSim/build_tarballs.jl

```diff
@@ -0,0 +1,35 @@
+using BinaryBuilder
+
+name = "JSBSim"
+version = v"1.1.6"
+
+# Collection of sources required to build JSBSim
+sources = [
+    GitSource("https://github.com/bcoconni/jsbsim.git", "a31ecb8727361aa74dee4ea7158c944848f78fb6"),
+]
+
+# Bash recipe for building across all platforms
+script = raw"""
+cd $WORKSPACE/srcdir/jsbsim
+mkdir build && cd build
+cmake -DCMAKE_INSTALL_PREFIX=$prefix -DCMAKE_TOOLCHAIN_FILE=${CMAKE_TARGET_TOOLCHAIN} -DCMAKE_BUILD_TYPE=Release -DBUILD_DOCS=OFF -DBUILD_PYTHON_MODULE=OFF ..
+make -j${nproc}
+cp julia/*JSBSimJL* $prefix/.
+cp ../julia/JSBSim.jl $prefix/.
+"""
+
+# Skip platforms with GCC 4.x because it does not support C++14
+platforms = [p for p in supported_platforms() if compiler_abi(p).gcc_version != :gcc4]
```

Comment:
I think you simply want to add the keyword argument `preferred_gcc_version=v"5"` to `build_tarballs` below

Reply:
Thanks ! I'll try that.